doc/_templates/edit_on_github.html

```diff
@@ -1,12 +0,0 @@
-<div role="note" aria-label="source link">
-    {% if sourcename is defined %}
-    {% set src = sourcename.split('.') %}
-    <h3>This Page</h3>
-    <ul class="this-page-menu">
-        <li>
-            <a href="https://github.com/{{ github_user }}/{{ github_repo }}/edit/{{ github_version }}/{{ doc_path }}/{{ pagename }}.{{ src[-2] }}">Edit this page</a>
-        </li>
-    </ul>
-     {% endif %}
-</div>
```

Comment:
Why remove the "edit on github" buttons? I find them useful to find the repo and file that needs editing

Reply:
it's not removed - this is a template from an old theme that we were manually inserting, the pydata theme has an "edit this page" button built into it (see the previewed docs)